### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by sekino** [oinume/tomahawk#57]
rised UnicodeEncodeError, when include non asciicode in command_output.

**Comment by kamatama41** [comment]
You should give more information, I think 😏 
- Runtime
- Output
- Steps to reproduce
- and so forth

**Comment by oinume** [comment]
@sekino 
Please let me know those information as @kamatama41 mentioned. Especially Python's version is important.

@kamatama41 
Thank you for following up!

**Comment by sekino** [comment]
@kamatama41 
It's very thoughtful of you. Thank you.

- runtime
```
$ tomahawk --version
tomahawk 0.7.3 with Python 2.6.6 (Linux-2.6.32-358.23.2.el6.x86_64-x86_64
-with-centos-6.4-Final)
```
- output
```
$ tomahawk -h <hosts> "echo 'あ'"
Traceback (most recent call last):
  File "/usr/bin/tomahawk", line 16, in <module>
    sys.exit(CommandMain(__file__).run())
  File "/usr/lib/python2.6/site-packages/tomahawk/base.py", line 76, in run
    return self.do_run()
  File "/usr/lib/python2.6/site-packages/tomahawk/command.py", line 59, in do_run
    return executor.execute(self.context.arguments)
  File "/usr/lib/python2.6/site-packages/tomahawk/command.py", line 217, in execute
    create_failure_last_message
  File "/usr/lib/python2.6/site-packages/tomahawk/base.py", line 262, in process_async_results
    output = create_output(color, output_format_template, command, host, exit_status, command_output)
  File "/usr/lib/python2.6/site-packages/tomahawk/command.py", line 188, in create_output
    'output': command_output,
  File "/usr/lib64/python2.6/string.py", line 205, in safe_substitute
    return self.pattern.sub(convert, self.template)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe3 in position 16: ordinal not in range(128)
```
- Steps to reproduce  
When non asciicode is included in the stdout of command which executed in tomahawk, it reproduce.
```
ex)
$ tomahawk -h <hosts> "echo 'あ'"
```  
I did not raised error in tomahawk 0.6.0 with Python 2.6.6

**Comment by oinume** [comment]
Thank you. I'm looking into it.

**Comment by oinume** [comment]
I found out the cause but it's very complex problem. Let me have much time to fix this.

**Comment by sekino** [comment]
Thank you for investigation of cause.
It's not a rush. I'm looking forward to fix

**Comment by oinume** [comment]
@sekino 
I fixed this issue and publish 0.7.4 to pypi. Please try it and let me know if you still have a problem.

**Comment by sekino** [comment]
@oinume 
Thank you for fixing.
I upgraded version and tried `tomahawk -h <hosts> "echo 'あ'"`, I did not have any problem.
